spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/repository/support/SpannerRepositoryImplTests.java

```diff
@@ -264,6 +265,20 @@ public void deleteByIdTest() {
 		verify(this.template, times(1)).delete(Object.class, A_KEY);
 	}
 
+	@Test
+	public void deleteAllByIdTest() {
+		List<Key> unconvertedKey = Arrays.asList(Key.of("key1"), Key.of("key2"));
+
+		when(this.entityProcessor.convertToKey(Key.of("key1"))).thenReturn(Key.of("key1"));
+		when(this.entityProcessor.convertToKey(Key.of("key2"))).thenReturn(Key.of("key2"));
+
+		new SimpleSpannerRepository<Object, Key>(this.template, Object.class)
+				.deleteAllById(unconvertedKey);
+		ArgumentCaptor<KeySet> argumentCaptor = ArgumentCaptor.forClass(KeySet.class);
+		verify(this.template).delete(eq(Object.class), argumentCaptor.capture());
+		assertThat(argumentCaptor.getValue().getKeys()).containsExactlyInAnyOrder(Key.of("key2"), Key.of("key1"));
+	}
```

Comment:
this `delete()` method itself already tested below.  
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/core/SpannerTemplateTests.java#L685-L698